src/webserver/WebServer.jl

```diff
@@ -6,19 +6,12 @@ import .PkgCompat
 
 include("./WebSocketFix.jl")
 
-# from https://github.com/JuliaLang/julia/pull/36425
-function detectwsl()
-    Sys.islinux() &&
-    isfile("/proc/sys/kernel/osrelease") &&
-    occursin(r"Microsoft|WSL"i, read("/proc/sys/kernel/osrelease", String))
-end
-
 function open_in_default_browser(url::AbstractString)::Bool
     try
         if Sys.isapple()
             Base.run(`open $url`)
             true
-        elseif Sys.iswindows() || detectwsl()
+        elseif Sys.iswindows() || detectwsl()  #= see PathHelpers.jl =#
```

Comment:
```suggestion
        elseif Sys.iswindows() || detectwsl()
```
Non-idiomatic Julia making it hard to read/understand

Reply:
Agreed!